docs/synthetics/bulk-add-monitors-api.md

```diff
@@ -0,0 +1,467 @@
+
+| Monitor Name | URL | Monitor ID |
+| --- | --- | --- |
+| bulk add 1 | https://example_1.com | 12345 |
+| bulk add 3 | https://example_3.com | 67891 | 
+
+Refer to [this link](https://sematext.com/docs/synthetics/using-the-api/#getting-the-apikey) to learn how to get monitor IDs, or you can use the Synthetics API to get all the available monitors for an app: [Get all monitors for an App](https://sematext.com/docs/synthetics/using-the-api/#get-all-monitors-for-an-app).
+
+- Then, you need to specify the monitor id within the endpoint URL.
+
+EU region:
+
+```javascript
+var endpoint = "https://apps.eu.sematext.com/synthetics-api/api/apps/" + appId + "/monitors/browser/" + id;
+```
+
+US region:
+
+```javascript
+var endpoint = "https://apps.sematext.com/synthetics-api/api/apps/" + appId + "/monitors/browser/" + id;
+```
+To do this, you need to extract the monitor ID from the new column you've added. Cut the line where we set the endpoint from outside the for loop and paste it inside to change the URL for each monitor.
+
+```javascript
+for (var i = 2; i <= sheet.getLastRow(); i++) {
+    var name = String(sheet.getRange(i, 1).getValue()).trim();
+    var url = String(sheet.getRange(i, 2).getValue()).trim();  
+    var id = String(sheet.getRange(i, 3).getValue()).trim();
+
+    var endpoint = "https://apps.eu.sematext.com/synthetics-api/api/apps/" + appId + "/monitors/browser/" + id;
+   
+    if (!name || !url || !id) {
+      Logger.log("Skipping row " + i + " due to missing data.");
+      continue;
+    }
+...
+```
+
+- Locate the ```locations``` array within your payload, add the new location.
+
+  ```javascript
+  "locations": [1, 2, 3]
+  ```
+Refer to [this link](https://sematext.com/docs/synthetics/using-the-api/#getting-the-locationid) to learn how to get location IDs.
+
+- Finally, change the API request method from POST to PUT, and then run the script.
+
+  ```javascript
+  var options = {
+      "method": "put",
+      "headers": {
+        "Authorization": "apiKey " + apiKey,
+        "Content-Type": "application/json"
+      },
+      "payload": JSON.stringify(payload),
+      "muteHttpExceptions": true
+    };
+  ```
+
+  ## Bulk Add Browser Monitors
+
+In the examples above, we bulk added HTTP monitors. You can follow the same steps to bulk add Browser monitors with minor changes in the Apps Script.
+
+The endpoint must be changed from **HTTP** to **Browser**.
+
+**Bulk Add Browser Monitors endpoint:**
+
+EU region:
+
+```javascript
+var endpoint = "https://apps.eu.sematext.com/synthetics-api/api/apps/" + appId + "/monitors/browser";
+```
+
+US region:
+
+```javascript
+var endpoint = "https://apps.sematext.com/synthetics-api/api/apps/" + appId + "/monitors/browser";
+```
+
+And, you must add ```"isPlaywright": true``` within your payload.
+
+```javascript
+ var payload = {
+      "name": name,
+      "interval": "5m",
+      "enabled": true,
+      "locations": [1, 2],
```

Comment:
Can I specify a bad location ID and mess things up?

We should probably say something about Private Locations being supported and adding them.

Reply:
If you specify a bad location ID you get validation error

> Response for row 2: {"success":false,"message":"invalid locations [190]"}

Added private locations.